codice-itest-impl/src/main/java/org/codice/itest/reporter/TestReporterConfiguration.java

```diff
@@ -52,4 +57,9 @@ public Consumer<TestResult> defaultLoggingDiagnosticTestReporter() {
     public Consumer<TestResult> exitCodeReporterConsumer(ExitCodeReporter exitCodeReporter) {
         return (tr) -> exitCodeReporter.register(tr.getTestStatus());
     }
+
+    @Bean("remainingTestsConsumer")
+    public Consumer<TestResult> remainingTestsReporterConsumer() {
+        return new RemainingTestsReporter(testNameList);
```

Comment:
✏️ We could get rid of the `testNamelist` variable and just do:
```suggestion
        return new RemainingTestsReporter(iTestConfigurationProperties.tests());
```